src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemPartition.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { FormGroup, Label, Radio } from '@patternfly/react-core';
+
+import { useAppDispatch, useAppSelector } from '../../../../store/hooks';
+import {
+  changeFileSystemPartitionMode,
+  selectFileSystemPartitionMode,
+} from '../../../../store/wizardSlice';
+
+const FileSystemPartition = () => {
+  const dispatch = useAppDispatch();
+  const fileSystemPartition = useAppSelector((state) =>
+    selectFileSystemPartitionMode(state)
+  );
+  return (
+    <FormGroup>
+      <Radio
+        id="radio-with-description"
+        label={
+          <>
+            <Label isCompact color="blue">
+              Recommended
+            </Label>{' '}
+            Use automatic partitioning
+          </>
+        }
+        name="radio-7"
```

Comment:
Could the radios get better descriptive names? Here maybe something like "fsc-radio-automatic"?